ansible/roles/vendor/requirements.yml

```diff
@@ -38,6 +38,9 @@
 - name: gsa.ansible-role-beats
   src: https://github.com/GSA/ansible-role-beats
   version: master
+- name: gsa.datagov-deploy-ci
+  src: https://github.com/GSA/datagov-deploy-ci
+  version: master
```

Comment:
If you want to add a tag to the master branch in this repo I will update this to point to it.